Automattic/sensei
# Add course navigation template setting

Fixes #4367

### Changes proposed in this Pull Request

* It introduces the course navigation template setting in the Course editor.
* It also updates the WP versions in the tests for the correct supported versions ([default value for meta was introduced in WP 5.5](https://make.wordpress.org/core/2020/08/04/registering-default-values-for-meta-data/)).

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_navigation', '__return_true' );`
* Create a new course.
* Make sure the default template for that option is the "Default Post Template".
* Edit an old course (created before this PR), and make sure "Default Post Template" is also the default option there.
* Save a different option, refresh the editor, and make sure it's persisted.


### Screenshot / Video

<img width="284" alt="Screen Shot 2021-11-01 at 17 01 38" src="https://user-images.githubusercontent.com/876340/139733980-7362fba5-e097-4d6c-8bff-f3185df1ef88.png">
